src/server/middleware/twitter.middleware.ts

```diff
@@ -83,11 +83,7 @@ export async function blockTwitterUsers(
   }
 
   const request = req.body as BlockTwitterUsersRequest;
-  const userCredential = firebase.auth.AuthCredential.fromJSON(
-    request.credential
-  ) as firebase.auth.OAuthCredential;
-
-  const response = await blockUsers(apiCredentials, userCredential, request);
+  const response = await blockUsers(apiCredentials, request);
```

Comment:
Just wanted to confirm that we don't need the fromJSON anymore? If I recall, there were fields that didn't translate correctly without it. See the (perhaps outdated/misplaced) comment at https://github.com/conversationai/harassment-manager/blob/9ab7aa4642040f51dda6613ab135b85693e6e17d/src/app/sheets_api.service.ts#L109 If that's still true we should move that comment here.

Reply:
Good catch! I tested the API calls and they succeeded, so it seems like we don't need `fromJSON` anymore. I removed the comment accordingly.

Reply:
Great, thanks for checking!